akka-actor-tests/src/test/scala/akka/actor/ExtensionSpec.scala

```diff
@@ -6,7 +6,7 @@ package akka.actor
 import java.util.concurrent.atomic.AtomicInteger
 
 import akka.testkit.EventFilter
-import akka.testkit.TestKit._
+import akka.testkit.scaladsl.TestKit._
```

Comment:
I keep the old one and deprecate it, so it breaks nothing before we decide to remove it.